server/src/services/cleanUserInput.js

```diff
@@ -0,0 +1,8 @@
+export const cleanUserInput = (formInput) => {
```

Comment:
Nice! Just for the practice, could you write up a quick jest test of this method?